css/main.css

```diff
@@ -5,133 +5,138 @@
     color: #1b1b1b;
 }
 
-a {
+.no-underline {
     text-decoration: none;
 }
 
 header {
-    text-align: left;
-    padding-inline: 10%;
+    position: relative;
+    padding: 0 max(50px, 10%);
     outline: 3px solid rgba(100, 100, 100, 0.459);
 }
 
-
 nav {
     height: 10vh;
+    max-height: 80px;
     line-height: 10vh;
     white-space: nowrap;
+    margin-left: 46%;
```

Comment:
Use `position: absolute;` and `top`/`right` attributes for nav positioning. But move out them to standalone class. `.nav` for `height` and `line-height` and `nav_positioning` for positioning of the block.